connect-inject/container_init.go

```diff
@@ -11,10 +11,14 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+// The init container is bounded in memory usage by the size of the consul binary,
+// So we will match the memory limit/requests to the size of the consul binary
```

Comment:
```suggestion
// So we will match the memory limit to the size of the consul binary plus 7Mi for safety buffer
```

Reply:
I think `125MB ~= max(size(consul-oss-binary), size(consul-ent-binary))`

Reply:
If we want the safety buffer, we'd need to add `7Mi` to the size of th enterprise binary.